java/vector/src/main/codegen/data/ValueVectorTypes.tdd

```diff
@@ -56,8 +56,10 @@
         { class: "Int", valueHolder: "IntHolder"},
         { class: "UInt4", valueHolder: "UInt4Holder" },
         { class: "Float4", javaType: "float" , boxedType: "Float", fields: [{name: "value", type: "float"}]},
-        { class: "IntervalYear", javaType: "int", friendlyType: "Period" }
-        { class: "Time", javaType: "int", friendlyType: "DateTime" }
+        { class: "DateDay", javaType: "int", boxedType: "Integer", friendlyType: "DateTime" },
```

Comment:
Another thing you can do is to have:
```
{class: "Date4", javaType: "int", boxedType: "Integer", friendlyType: "DateTime"}
{class: "Time4", javaType: "int", boxedType: "Integer", friendlyType: "DateTime"}
{class: "Time8", javaType: "long", boxedType: "Long", friendlyType: "DateTime"}
{class: "Timestamp8", javaType: "long", boxedType: "Long", friendlyType: "DateTime"}
```
Here and have a specialized constructor for date/time/timestamp types that takes a time unit parameter.

I am not sure which way is better.